README.md

```diff
@@ -153,6 +153,14 @@ The latter line should print an instance of the event `ObjectEvent` which is cre
 This instance has many methods which can be used to view or modify the EPCIS 2.0 event.
 You can view all of them in ./src/entity/events/Event.js.
 
+It is now possible to validate EPCIS events without wrapping them in an EPCIS Document.
+
+```js
+const eventList = epcisDocument.getEventList();
```

Comment:
I would add an event that doesn't come from an EPCISDocument.
Just create a new ObjectEvent for the example. Otherwise they won't understand the purpose of validating it.